index.html

```diff
@@ -301,5 +291,30 @@ <h2 class="section2-h2">skills</h2>
         </li>
       </ul>
     </section>
+
+    <!-- footer -->
+
+    <footer>
+      <h1 class="contact_title">Contact me</h1>
+      <p class="contact_paragraph">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it!
+      </p>
+      <form class="form-section">
+        <label for="name">name</label>
+        <input type="text" id="name" placeholder="enter your name" required />
+
+        <label for="email">email</label>
+        <input
+          type="email"
+          id="email"
+          placeholder="enter your email"
+          required
+        />
+        <textarea id="message">Write your message here.</textarea>
```

Comment:
Kindly remove the `textarea` tags and add them as placeholder text instead.  Also, please add a `max-length=500` to the `textarea` and the `required` attribute as` specified in the project description. (see image below)
<img width="446" alt="Screenshot 2023-03-29 at 15 14 43" src="https://user-images.githubusercontent.com/107192076/228680462-1e382bb6-f91a-40b9-b678-66442be10256.png">